From b48b038431a65f2b79125f1fa56762ee826f9333 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 24 Aug 2018 07:48:42 -0600
Subject: [PATCH 8/8] Revert "Disable rild from starting at power up"

* Ain't nobody wants to have rild started from a separate shell
  script in init like CAF wants to do in:
  https://source.codeaurora.org/quic/la/device/qcom/common/tree/rootdir/etc/init.class_main.sh?h=LA.UM.7.2.r1-04000-sdm660.0

This reverts commit 8eb100eb4a79d60e8c441ab72c270167ed8b67b3.

Change-Id: I3e7eae85b851c8cd3d0b8ca7a6ea9308c4f22c47
---
 rild/rild.legacy.rc | 1 -
 rild/rild.rc        | 1 -
 2 files changed, 2 deletions(-)

diff --git a/rild/rild.legacy.rc b/rild/rild.legacy.rc
index 6f2f32a..b27e1da 100644
--- a/rild/rild.legacy.rc
+++ b/rild/rild.legacy.rc
@@ -1,6 +1,5 @@
 service ril-daemon /vendor/bin/hw/rild
     class main
     user radio
-    disabled
     group radio cache inet misc audio log readproc wakelock
     capabilities BLOCK_SUSPEND NET_ADMIN NET_RAW
diff --git a/rild/rild.rc b/rild/rild.rc
index 5178fa3..f6beb54 100644
--- a/rild/rild.rc
+++ b/rild/rild.rc
@@ -1,6 +1,5 @@
 service vendor.ril-daemon /vendor/bin/hw/rild
     class main
     user radio
-    disabled
     group radio cache inet misc audio log readproc wakelock
     capabilities BLOCK_SUSPEND NET_ADMIN NET_RAW
-- 
2.17.1

